### ISSUE TITLE
[FEAT] Improvements

### ISSUE THREAD

**Issue by muuvmuuv** [muuvmuuv/portfolio#65]
In order to have less hassle with my portfolio I want to do a few things:

- [ ] Use Next.js because Gatsby feels very bloated and is missing features like SSG
- [ ] Remove docker-compose, its not needed with Next.js and Vercel
- [ ] Update Vercel deployment
- [ ] Update linter and formatter configurations
- [ ] Auto switch theme based on system theme (no config)
- [ ] Remove breadcrump
- [ ] Page transitions
- [ ] Remove tests... not needed
- [ ] Use `standard-version` for releases
- [ ] Change Now to Vercel
- [ ] Update GH Actions

**Comment by muuvmuuv** [comment]
Coming in v6
